uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_calendar-feeds.scss

```diff
@@ -1,3 +1,9 @@
+calendar-feeds-dropdown-button {
+  .dropdown-menu .divider {
+    margin: 0;
```

Comment:
@igorpavlov I'm a bit worried about removing the margin here. If the mockup has no margin, should it be the same for every other dropdown with a divider? Why is this an exception? I think we need to agree what's the rule here and apply it globally or just remove this and let Bootstrap do its thing.

Reply:
can you show me here screenshots of what it looks like now in this PR and what it look like if we remove this exception and let Bootstrap do its thing

Reply:
Without extra styling:

![image](https://user-images.githubusercontent.com/3973243/42280716-70329006-7f99-11e8-9586-4d574971763c.png)

With extra styling:

![image](https://user-images.githubusercontent.com/3973243/42280757-8ae59178-7f99-11e8-979a-78ac01514196.png)

Reply:
Decision: SSP - remove custom styling

Reply:
We have custom styling in JobContract:

```css
#hrjob-contract .dropdown-menu .divider {
  height: 1px;
  margin: 8px 0;
  overflow: hidden;
  background-color: #e5e5e5;
}
```

I don't think the SSP styles will clash with ^.

However, the only place where there is a **markup** with a divider is in reqangular `ui-select` directive. So I think we are safe.

Reply:
After styles are removed from SSP theme the dropdown looks like this:

![image](https://user-images.githubusercontent.com/3973243/42284160-88ea887e-7fa3-11e8-9f89-3287811ca4e7.png)